lib/pbench/server/api/resources/query_apis/datasets_list.py

```diff
@@ -120,30 +120,29 @@ def postprocess(self, es_json: Dict[AnyStr, Any]) -> Dict[AnyStr, Any]:
         datasets = []
         hits = es_json["hits"]["hits"]
         self.logger.info("{} controllers found", len(hits))
+        controller = None
         for dataset in hits:
             src = dataset["_source"]
             run = src["run"]
+            if controller:
+                if controller != run["controller"]:
+                    self.logger.warning(
+                        "Expected all controllers to match: {} and {} don't match",
+                        controller,
+                        run["controller"],
+                    )
+            else:
+                controller = run["controller"]
```

Comment:
If I might suggest....
```
            if not controller:
                controller = run["controller"]
            elif controller != run["controller"]:
                self.logger.warning(
                    "Expected all controllers to match: {} and {} don't match",
                    controller,
                    run["controller"],
                )
```
This saves a level of indentation and a line of screen realestate.

Reply:
Sure.